From 3973d207d50eb47ed37e5e18173063c389e69e88 Mon Sep 17 00:00:00 2001
From: your name <your_email@your_domain.com>
Date: Sun, 31 Mar 2019 12:20:33 +0000
Subject: [PATCH] update

---
 Android.bp     | 1 +
 product/gps.mk | 5 ++++-
 2 files changed, 5 insertions(+), 1 deletion(-)

diff --git a/Android.bp b/Android.bp
index fe6d1d6..0c76aab 100644
--- a/Android.bp
+++ b/Android.bp
@@ -1,3 +1,4 @@
 subdirs = [
+    "gps",
     "usb",
 ]
diff --git a/product/gps.mk b/product/gps.mk
index 1608d9c..1ec420b 100644
--- a/product/gps.mk
+++ b/product/gps.mk
@@ -19,4 +19,7 @@ PRODUCT_COPY_FILES += \
     $(LOCAL_PATH)/configs/gps/gps_debug.conf:system/etc/gps_debug.conf
 
 PRODUCT_PACKAGES += \
-    android.hardware.gnss@1.0-impl
+	android.hardware.gnss@1.0-impl.hero \
+	android.hardware.gnss@1.0-service \
+	android.hidl.base@1.0 \
+	android.hidl.manager@1.0
